Source/Documentation/Manual/cabs.rst

```diff
@@ -403,6 +403,59 @@ Here is an example of use of the odometer control blocks within a .cvf file::
     Units ( FEET )
   )
 
+.. _cabs-distributed-power:
+
+Distributed Power
+-----------------
+
+The principles of Distributed Power are described :ref:`here <distributed-power>` .
+
+Distributed Power data can be displayed using control ORTS_DISTRIBUTED_POWER. Here 
+an example of use::
+
+	ScreenDisplay (
+      Type ( ORTS_DISTRIBUTED_POWER SCREEN_DISPLAY )
+      Position (  164.4 286.5 136 52 )
+		Parameters (
+         FullTable True
+		)
+      Units ( KM_PER_HOUR )
+		ORTSDisplay ( 1 )
+		ORTSScreenPage ( "2300-0" )
+   )  
+
+Here below an example of the output of the above control.
+
+.. image:: images/cabs-distributed-power.png
```

Comment:
Image seems to be missing from the commit

Reply:
Missing picture added, thanks